runtime/doc/options.txt

```diff
@@ -2835,15 +2835,16 @@ A jump table for the options with a short description can be found at |Q_op|.
 			blinking off you can use "blinkon0".  The cursor only
 			blinks when Vim is waiting for input, not while
 			executing a command.
-			To make the cursor blink in an xterm, see
-			|xterm-blink|.
+			To make the cursor blink in an xterm, see |xterm-blink|.
 		{group-name}
 			a highlight group name, that sets the color and font
-			for the cursor
+			for the cursor.
+			Requires 'termguicolors' to be set.
```

Comment:
Yes. Though in the future we could support cterm. 

In general I would rather leave it at "guicursor will try its best to work in the TUI". Why do we need to spell out every single case where it might work? There's probably dozens or hundreds of possible combinations of environmental factors. Who cares? It's a cursor. Either it works or it doesn't. Let's continue to try to make it work better, but passively.